source/features/parse-backticks.tsx

```diff
@@ -25,6 +25,10 @@ function init(): void {
 		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalDiscussionList`
 		'[data-channel^="check_suites"] a', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a:first-child', // `isRepoRoot` ("Repository refresh" layout)
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden pre', // `isRepoRoot` ("Repository refresh" layout)
```

Comment:
I already see it on `master` 
<img width="890" alt="" src="https://user-images.githubusercontent.com/1402241/86266404-74c15480-bbc5-11ea-88c9-0df9ac86e996.png">

Reply:
These are the commit message and description here:
![image](https://user-images.githubusercontent.com/44045911/86267406-3c3b6e00-bbf9-11ea-94e6-1096bda35edf.png)